src/binary-writer.cc

```diff
@@ -1002,8 +1009,12 @@ Result BinaryWriter::WriteModule(const Module* module) {
   }
 
   if (module->excepts.size()) {
-    // TODO(karlschimpf) Define.
-    WABT_FATAL("write exception section not implemented");
+    BeginCustomSection("exception", LEB_SECTION_SIZE_GUESS);
+    write_u32_leb128(&stream_, module->excepts.size(), "exception count");
```

Comment:
Is exception not going to be a buildin section?  (sorry I should really just read the spec..)